qcodes/tests/dataset/test_dataset_export.py

```diff
@@ -675,6 +714,75 @@ def test_export_2d_dataset(
     assert xr_ds.identical(xr_ds_reimported)
 
 
+def test_export_numpy_dataset(
+    tmp_path_factory: TempPathFactory, mock_dataset_numpy: DataSet, caplog
+) -> None:
+    tmp_path = tmp_path_factory.mktemp("export_netcdf")
+    mock_dataset_numpy._export_limit = 0
+    with caplog.at_level(logging.INFO):
+        mock_dataset_numpy.export(export_type="netcdf", path=tmp_path, prefix="qcodes_")
+
+    assert (
+        "Dataset is expected to be larger that threshold. Using distributed export."
+        in caplog.records[0].msg
+    )
+    assert "Writing individual files to temp dir" in caplog.records[1].msg
+    assert "Combining temp files into one file" in caplog.records[2].msg
+    assert "Writing netcdf file using Dask delayed writer" in caplog.records[3].msg
+
+    loaded_ds = xr.load_dataset(mock_dataset_numpy.export_info.export_paths["nc"])
+    assert loaded_ds.x.shape == (10,)
+    assert_allclose(loaded_ds.x, np.arange(10))
+    assert loaded_ds.y.shape == (11,)
+    assert_allclose(loaded_ds.y, np.arange(10, 21, 1))
+
+    arrays = []
+    for i in range(10):
+        arrays.append(np.arange(10 + i, 21 + i))
+    expected_z = np.array(arrays)
+
+    assert loaded_ds.z.shape == (10, 11)
+    assert_allclose(loaded_ds.z, expected_z)
+
+    _assert_xarray_metadata_is_as_expected(loaded_ds, mock_dataset_numpy)
+
+
+def test_export_numpy_dataset_complex(
+    tmp_path_factory: TempPathFactory, mock_dataset_numpy_complex: DataSet, caplog
+) -> None:
+    tmp_path = tmp_path_factory.mktemp("export_netcdf")
+    mock_dataset_numpy_complex._export_limit = 0
```

Comment:
What do you think about having 1 test that asserts that with the currently-set export limit we are NOT using multi-file/delayed-dask export? So that we know that simple export is also working.

Reply:
sure no harm in that